tests/integration/test_submit.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from click.testing import CliRunner
+
+from cromshell.__main__ import main_entry as cromshell
+
+
+class TestSubmit:
+    @pytest.mark.parametrize(
+        "wdl, json, exit_code",
+        [
+            ("tests/workflows/helloWorld.wdl", "tests/workflows/helloWorld.json", 0),
+            ("tests/workflows/helloWorld.wdl", "tests/workflows/not_valid.json", 1),
+            ("tests/workflows/not_valid.wdl", "tests/workflows/helloWorld.json", 1),
+        ]
+    )
+    def test_submit(self, local_cromwell_url: str, wdl: str, json: str, exit_code: int):
+        runner = CliRunner(mix_stderr=False)
+        result = runner.invoke(
+            cromshell,
+            [
+                "--cromwell_url",
```

Comment:
This is a personal style thing that probably contradicts PEP-8, but when building command lines like this I like to put the arguments on the same line as the value.  I think it makes it easier to read.

ex:
```
"--cromwell_url", local_cromwell_url,
"submit",
wdl,
json
```

Reply:
tox linting doesn't like it when they are on the same line